source/tests/pt/model/test_pairtab_atomic_model.py

```diff
@@ -114,9 +114,8 @@ def test_cross_deserialize(self):
         expected_result = self.model.forward_atomic(
             self.extended_coord, self.extended_atype, torch.from_numpy(self.nlist)
         )
-
         np.testing.assert_allclose(
-            result["energy"], expected_result["energy"], 0.0001, 0.0001
+            result["energy"], expected_result["energy"].detach().numpy(), 0.0001, 0.0001
```

Comment:
you may use to_numpy_array in pt/utils/utils.py, which takes care of device transfer.